scripts/capa_as_library.py

```diff
@@ -16,7 +16,7 @@
 from capa.engine import *
 
 
-# == Render ddictionary helpers
+# == Render dictionary helpers
```

Comment:
It's possible this is a thing. I think at one point I convinced myself it was. But I've given up...